server/domain/academic-record/models/CourseEnrollments.ts

```
@@ -0,0 +1,55 @@
+export class CourseEnrollments {
+  constructor(private readonly enrollments: CourseEnrollment[]) {}
+
+  static create(enrollments: CourseEnrollment[]): CourseEnrollments {
+    return new CourseEnrollments(enrollments);
+  }
+
+  getAll(): CourseEnrollment[] {
+    return [...this.enrollments];
+  }
+
+  getBySemester(year: number, semester: number): CourseEnrollments {
```

ESLint

[error] 'year' is defined but never used.

(@typescript-eslint/no-unused-vars)